lib/oli_web/live/sections/edit_view.ex

```diff
@@ -92,6 +104,23 @@ defmodule OliWeb.Sections.EditView do
     end
   end
 
+  def handle_event("save_labels", %{"view" => params}, socket) do
+    socket = clear_flash(socket)
+
+    params = Map.merge(%{"unit" => "unit", "module" => "module", "section" => "section"}, params, fn _k, v1, v2 ->
```

Comment:
I noticed when changing only a single value of these three, the other ones are reset to the default value. This is most likely because the text feild value is empty with the placeholder being set to the value. I think an easy fix here would be to set the value of the text field to the field value when the textfield renders, reserving the placeholder for only when the value is actually nil

Reply:
Set empty text field to default value